src/components/FullDetailView/FullDetailPage.js

```diff
@@ -98,8 +98,13 @@ export class FullDetailPage extends Component {
   previewDefinition(nextComponent) {
     const { token, component, uiCurateGetDefinitionPreview } = this.props
     const { changes } = this.state
-    if (!component && !nextComponent) return false
-
+    if (
+      (!component || isEmpty(component.changes)) &&
+      (!nextComponent || isEmpty(nextComponent.changes)) &&
+      isEmpty(changes)
+    )
+      return false
+    console.log(component, nextComponent, changes)
```

Comment:
do we need to keep the `console.log` statement?

Reply:
Of course we don't. Just updated the PR removing this instruction.